modules/react/collection/spec/useListLoader.spec.ts

```diff
@@ -0,0 +1,70 @@
+import {getPageNumber, getPagesToLoad} from '../lib/useListLoader';
+
+describe('getPageNumber', () => {
+  it('should return 1 when the index is 0', () => {
+    expect(getPageNumber(0, 0)).toEqual(1);
+  });
+
+  it('should return 1 if the index is less than the page size', () => {
+    expect(getPageNumber(5, 10)).toEqual(1);
+  });
+
+  it('should return 1 if the index is one less than page size', () => {
+    expect(getPageNumber(9, 10)).toEqual(1);
+  });
+
+  it('should return 2 if the index is the same as page size', () => {
+    expect(getPageNumber(10, 10)).toEqual(2);
+  });
+});
+
+describe('getPagesToLoad', () => {
+  it('should load page 1 when no data is loaded and the loading buffer', () => {
```

Comment:
Is there something missing after "...and the loading buffer"?

Reply:
"... is 3"